set.mm

```diff
@@ -426182,18 +426182,17 @@ number of vertices in a friendship graph is (k(k-1)+1), see
     conventions.1 $e |- ph $.
     $(
        <HTML>
-       <p>
-       Here are some of the conventions we use in the Metamath Proof Explorer
-       (MPE, set.mm), and how they correspond to typical textbook language
-       (skipping the many cases where they are identical).
+       <p>Here are some of the conventions we use in the Metamath Proof
+       Explorer (MPE, set.mm), and how they correspond to typical textbook
+       language (skipping the many cases where they are identical).
        For more specific conventions, see:
        <ul>
        <li> ~ conventions-labels : conventions related to labels</li>
        <li> ~ conventions-comments : conventions related to comments</li>
        </ul>
 
        <ul>
-       <li><b>Notation.</b>
+       <li><p><b>Notation.</b>
```

Comment:
What is the `<p>` here for? Are we better off putting some style on the `<li>`?

Reply:
(I recommend viewing this in a browser.) It adds paragraph breaks between the list items. We could do it with css, e.g. `style="margin-bottom:5px"` on the `li`, but some of the list items are multiple paragraphs so this also starts interacting with the global styles.

I want to majorly overhaul the way styling is done in metamath and set.mm, but the first step of that is to make mm-web-rs the official web site generation mechanism, so that we can drop all the regex HTML munging.